components/log-viewer-webui/server/src/fastify-v2/app.ts

```diff
@@ -72,7 +72,9 @@ export default async function serviceApp (
             "Unhandled error occurred"
         );
 
-        if ("undefined" !== typeof err.statusCode && INTERNAL_SERVER_ERROR_CODE > err.statusCode) {
+        if ("undefined" !== typeof err.statusCode &&
+            StatusCodes.INTERNAL_SERVER_ERROR as number > err.statusCode
```

Comment:
would it be better to write
```suggestion
            Number(StatusCodes.INTERNAL_SERVER_ERROR) > err.statusCode
```